### PR TITLE
[Translation into ja] Feature/translation nightly build into ja

### PR DESCRIPTION
**Pull request by uraway**
ref: https://github.com/microsoft/TypeScript-Website/issues/220

Translation Nightly Builds.md in to ja

diff: https://github.com/microsoft/TypeScript-Website/commit/3e217634258b9d5329d0a97d06c0ccb6dbe3651a

### CODE REVIEW

File: packages/documentation/copy/ja/Nightly Builds.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+"typescript_tsdk": "<path to your folder>/node_modules/typescript/lib"
+```
+
+詳細は[TypeScript Plugin for Sublime Text installation documentation](https://github.com/Microsoft/TypeScript-Sublime-Plugin#installation)を参照してください。
+
+### Visual Studio 2013 と 2015
+
+> 注意: ほとんどの変更は新しいバージョンのVS TypeScriptプラグインをインストールする必要はありません
+
+現在、ナイトリービルドには完全なプラグインのセットアップは含まれていませんが、インストーラーも同様に毎晩公開する作業を行っています。
```
Comment (sasurau4) [inline packages/documentation/copy/ja/Nightly Builds.md]: この、nightlyはおそらく単語の意味通りではなく、nightly buildの意味だと思います（たぶん）
```suggestion
現在、ナイトリービルドには完全なプラグインのセットアップは含まれていませんが、ナイトリービルドベースのインストーラーも同様に公開する作業を進めています。
```

**Review by sasurau4 — CHANGES_REQUESTED** [review]
1点だけ指摘しました　:+1:

File: packages/documentation/copy/ja/Nightly Builds.md (hunk 1 above)
Comment (uraway) [reply packages/documentation/copy/ja/Nightly Builds.md]: ありがとうございます！

**Review by sasurau4 — APPROVED** [review]
LGTM :+1:

**Comment by sasurau4** [comment]
@orta hmm, Codeowners merging failed. Could you look it?

**Comment by orta** [comment]
Thanks!

**Pull request merged**